core/blockchain.go

```diff
@@ -360,6 +361,14 @@ func (bc *BlockChain) CurrentFastBlock() *types.Block {
 	return bc.currentFastBlock.Load().(*types.Block)
 }
 
+// SetProcFeedback adds a feedback channel where true is sent each time block
```

Comment:
Instead of the single channel, there should be a `event.Feed` for block processing start/end events. These events are useful in multiple places. For example, chain indexers could also be paused while block processing runs.

Reply:
done